### DISCUSSION TITLE
Indicator on the Pedal page to show activity

### DISCUSSION THREAD

**Discussion by Jailin** [alf45tar/PedalinoMini#447] [Ideas]
Basically a fake LED to show that the pedal/button/switch is working. Would help with troubleshooting.

**Comment by jumperrg** [comment]
I'm not sure if it's gonna work, or be efficient. The web page loading time is quite long, bc it's loaded via HTTP, so it's not a real time thing.

**Reply by alf45tar** [reply]
It can be real time too using websocket.
